crates/consensus/src/consensus/mod.rs

```diff
@@ -111,6 +111,7 @@ impl Stage {
             .or_panic()
     }
 
+    #[instrument(level = Level::INFO, skip(self))]
```

Comment:
This probably has nothing to do here?

Reply:
No no, it does. I wanted to see this one as log while working on this 😅 ... as surprising as it may seem. 

I separated the change in a single commit: 3df064bf70e635a08b57a284d37fc85842bc92bf